site/en/reference/cli/concepts/netdata.md

```diff
@@ -29,6 +32,19 @@ unpublish
 Done
 ```
 
+### `full` commands
```

Comment:
We probably need to run the OT reference to get this command into the site, and then perhaps come up with a scenario where this is used in this guide.  I.e. when would you use it and what for?

You can ask @abtink about it.